charts/lighthouse/values.yaml

```diff
@@ -50,7 +50,7 @@ image:
   parentRepository: ghcr.io/jenkins-x
 
   # image.tag -- Docker images tag
-  tag: 0.0.750
+  tag: 1.1.0
```

Comment:
note that this field is automatically updated for each tag - see https://github.com/jenkins-x/lighthouse/blob/v1.1.0/charts/lighthouse/values.yaml#L53 for example

Reply:
So, the tag does not update automatically on the main branch. Right? It's reasonable if I understand it correctly. But errors happened if someone wants to install the Lighthouse chart from the main branch.

I know that it's not necessary to update this tag from the main branch. It would better if it's an existing tag.

So, what is your suggestion?

Reply:
well, you can update the main branch, but in this case we should do it automatically, such as what we're doing in the jx-release-version repo: https://github.com/jenkins-x-plugins/jx-release-version/commit/fdc1294222fa477caf96c9684ec64d2a2db344c5

but how do you install the chart from the main branch? (or more important "why")
the recommended way to install it is from the official chart repo at https://storage.googleapis.com/jenkinsxio/charts so that you can specify a version.

Reply:
if you want to update a file as part of the release pipeline, you can do something like https://github.com/jenkins-x-plugins/jx-release-version/commit/303a9eaa773677e794385b1e93651145a5a421e5

but for your use-case, I think it will be easier/faster to just install the chart from your own "fixed" branch - until the James are back and merge your fix

Reply:
using `latest` is maybe the safest in the  main branch?

Reply:
Consider the version from a specific tag is correct.  Using `latest` might be a safe way. I'm going to take your suggestion.

Reply:
generally, installing charts using actual versioned charts is much better anyway really - than using whatever the source in git in the main branch is

Reply:
Yes, I agree with you @jstrachan . But it would be better if we can keep all branches work well.

Reply:
actually we're not tagging releases with `latest` so thats not an option yet

Reply:
Or,  there's an alternative solution is that pushing the images with tag `0.0.750` from `gcr.io` to `ghcr.io`.

Reply:
ok I've added an extra patch to publish both the specific version and the `latest` tag so we should be good now